### PR TITLE
Solo project Submission

### PR DESCRIPTION
**Pull request by kelibst**
Solo project Submission.
Submitting my solo project for review

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,32 @@
...
+![demopage](demopage.png)
+
+## Built With
+
+- HTML5 
+- CSS3
+
+## Live Demo
+
+[Live Demo Link](https://raw.githack.com/kelibst/appleStore/tree/featured-branch/index.html)
```
Comment (MauricioRobayo) [inline README.md]: The link is not working, it should be: https://raw.githack.com/kelibst/appleStore/featured-branch/index.html

**Review by MauricioRobayo — CHANGES_REQUESTED** [review]
Hi!

Great work so far 😄, but need to fix some issues:

- [ ] Fix [HTML validation errors](https://validator.w3.org/nu/?doc=https%3A%2F%2Fcdn.statically.io%2Fgh%2Fkelibst%2FappleStore%2Ffeatured-branch%2Findex.html).
- [ ] Navigation bar is not displaying correctly:
image
- [ ] Footer menu is not diplaying correctly:
image

This is the live preview I used for your project as the link on your README.md was not working: https://cdn.statically.io/gh/kelibst/appleStore/featured-branch/index.html

Status: **Request changes**

File: README.md (hunk 1 above)
Comment (kelibst) [reply README.md]: Thanks. I will make the necessary changes.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <script src="https://kit.fontawesome.com/5e0cee5c01.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="public/css/reset.css">
+    <link rel="stylesheet" href="public/css/style.css">
+    <title>Apple</title>
+</head>
+
+<div>
+
+   <div class="container-fluid">
```
Comment (euqueme) [inline index.html]: [Optional] instead of using \<div\> change this to \<main\>

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request under the review by Maria Eugenia Quemé P. (TSE)

### Hello @kelibst :handshake: 

### Great job so far I think you almost nailed the gradient in the navbar so claps :clap:  to you. thank you for doing the requested changes but there are still a couple of things that called my attention:

- [ ] Add a separator to each button in the navbar (perhaps a border-right) so it looks even more similar to the orignial

- Yours
image

- Original
image

[Optional] I highly encourage the use of the \<main\> tag for the dominant content of the \<body\> of a document I added a comment in your code where I think it should be placed.

### Thats it! :raised_hands: its been a blast reviewing this project :guitar:  please :pray: make the change :memo:  and request again for a code review :rocket: 

Happy Coding! :woman_technologist: 

STATUS OF THE PROJECT: REQUESTED CHANGES :traffic_light:

**Review by mosesogwo — CHANGES_REQUESTED** [review]
**Hi @kelibst, Nice work :+1: , just a little change here:**
1. It seems our footer got distorted while you were trying to make the requested changes
image

2. We have a horizontal scroll bar on the page, can we try to fix it?

In future projects, please describe the **content** of the project/branch in your pull request title and summary and not as you have it now.

_**Kindly make these changes and submit another code review request.**_

**Cheers:smile:**

**Status: Changes requested.**

**Comment by kelibst** [comment]
Thanks *@mosesogw, Sorry about the distortion. let me fix that quickly*

**Review by mosesogwo — APPROVED** [review]
**:trophy:Congratulations @kelibst, your project is now approved.**

Before you merge to master, specify the width of your .hero class in percentage to remove the horizontal scroll.

**Good luck in your future projects:smile:**

**Status: Approved!**

**Pull request merged**